ci/lava/tests/test-build-test-images.py

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env python3
```

Comment:
Again - maybe better to refer to as tutorial images? Or at least application images or something. Test images seems wrong, unless you think we are going to increase scope to boot loaders etc.

Reply:
changed.